src/Microsoft.OData.Client/BaseSaveResult.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.OData.Client
     using System.Text;
     using System.Threading;
     using Microsoft.OData;
+    using Microsoft.OData.Client.Materialization;

```

Comment:
remove unused import. 
```suggestion
    using Microsoft.OData.Client.Materialization;
remove unused import . 
```